api/api.js

```diff
@@ -22,12 +22,9 @@ import * as solanaWeb3 from '@solana/web3.js';
 
 import {FriendlyGet} from './friendlyGet';
 import config from './config';
-
-//
-// FIXME: make configurable
-//
-//const FULLNODE_URL = 'http://beta.testnet.solana.com:8899';
-const FULLNODE_URL = 'http://localhost:8899';
+import {addNetworkExplorerRoutes} from './network-explorer';
+import {addLegacyRoutes} from './legacy-routes';
+import {FULLNODE_URL} from './fullnode-url';
```

Comment:
(these are just moving code into more specific places for better organization/removal as appropriate)